pkg/auth/model/model.go

```diff
@@ -134,6 +134,7 @@ type Group struct {
 	ID          string    `db:"id"`
 	CreatedAt   time.Time `db:"created_at"`
 	DisplayName string    `db:"display_name" json:"display_name"`
+	Description *string   `db:"description" json:"description"`
```

Comment:
Why not make this be a string and have the default be ""? That way it will be consistent with the other values. Also, there isn't really a different meaning to a nil description or an empty string description

Reply:
Oh ok